README.md

```diff
@@ -1,248 +1,417 @@
-Take your pick from the following:
+We have learned from our mistakes and _documented_ most of them!
+Hopefully it's ***useful*** to other people.
 
-- "*We don't have time to write tests, we just need to get this done Yesterday!*"
-- "*Testing takes too long and means we won't deliver as many features...*"
-- "*Its just a small fix, we don't need to write a test*..."
-- "*I'm a developer, I don't write tests, we need to hire a tester...*"
-- "*If I write a test for every bit of code, then I have to update the tests when I need to change the code, so it's twice as much work...*"
+### What can we _Make_ with the Knowledge/Skills in the Book?
 
-and, last but not least:
+The knowledge/skills you will gain from learning/practicing
+the content will allow you to build web-based applications (_that "**Work Everywhere**"_)
+for *many* reasons including (to):
 
-- "*That's the way we have always done it here...*"
++ ***Connect & communicate*** with our friends, colleagues & communities
++ ***Record & share*** information, knowledge, news, events, etc.
++ **Manage** all aspects of our lives (calendars, task/time trackers,
+  shopping/wish lists, etc.)
++ **Run** organisations/businesses (_Administration, Workflow & "Management" Apps_)
++ **Buy/sell** things (_E-Commerce_)
++ ***Have fun***! (_e.g. online games! augmented/virtual reality!_)
 
-As a comparison, [***this***](http://en.wikipedia.org/wiki/Nokia_8110) is how we ***used*** to do Mobile Phones ... How many people do you still see carrying "*The Matrix Phone*" (*which was all-the-rage in the* ***90's***!) *today*?
+Most universities/schools (_still_) don't teach practical web skills
+as the *foundation* for computer science / software engineering ...
+there are a number of _reasons_ for this, but the primary reason is that
+the learning institution's curriculum is calcified by inertia...
 
-![Matrix Phone Nokia 8110](https://raw.github.com/nelsonic/practical-js-tdd/master/manuscript/images/00-matrixphone.jpg "Matrix Phone Nokia 8110")
+The result of people not learning "modern" web technologies at "school" is
+the _majority_ of companies/organisations *still* lack skills so most apps suck!
+We are "_fixing_" the "_problem_".
 
-![Steve Jobs introducing iPhone in 2007](https://raw.github.com/nelsonic/practical-js-tdd/master/manuscript/images/00-steve-jobs-introduces-iphone.jpg "Steve Jobs introducing iPhone in 2007")
+<!--
+If you are _still_ not writing tests _**why not**_?
+We _urge_ you to _read_ the *first* chapter ("_A Brief Introduction to TDD_") and decide for yourself if you _still_ think its "_too difficult_" or "_takes longer_".
+-->
 
+### "_Free_" and Open Source...?
 
-We have better phones; shouldn't we also have better software development methods?
+Yes, the book is **100% _Free_ and _Open Source_**  on GitHub for anyone to
+read and [contribute](#contribute) to!    
+We (*strongly*) believe that ***all learning materials*** should be
```

Comment:
I know what you mean by this, but we don't want to give the impression (given the massive library we have) that we expect people to pour months of their life into creating quality content for *free*.
We need to convey that we know and acknowledge the value of people's time (which is why we pay github even though all our repos are open source for example) whilst still detailing our reasons for open sourcing ours.
I know there's a little on this further down, but this is the kind of sentence that causes a reaction big enough that people will stop reading and just *whine.*